source/use-cases/sensitive-data-encryption.txt

```diff
@@ -516,6 +529,30 @@ MedcoMD engineers write a function to create a new patient record:
                  .append("insurance", insurance);
              collection.insertOne(patient);
          }
+   .. tab::
+      :tabid: nodejs
+      
+      .. code-block:: javascript
+      
+         function insertPatient(collection, name, ssn, bloodType, medicalRecords,  policyNumber, provider) {
+             collection.insertOne({
+             name: name,
```

Comment:
Nitpick: this block should be indented one more level than its parent. For automatic javascript formatting, you can use the jslint npm module.

Reply:
Fixed